CMakeLists.txt

```diff
@@ -38,15 +38,9 @@ if(TARGET run-validate)
 endif()
 
 option(BUILD_TESTING "Enable testing" ON)
+option(STL_BUILD_BENCHMARKING "Enable benchmarking" OFF)
 set(VCLIBS_SUFFIX "_oss" CACHE STRING "suffix for built DLL names to avoid conflicts with distributed DLLs")
 
-set(CMAKE_CXX_FLAGS "")

```

Comment:
I would prefer if we could move those cmake simplifications into their own PR

Reply:
We can't, because we need the benchmarks to not get these. This whole PR is about setting up the build system such that we can build benchmarks.